docker-compose.yaml

```diff
@@ -0,0 +1,30 @@
+version: '3.5'
+
+services:
+  app:
+    image: node:14.15.4
+    ports:
+      - 3333:3333
+    container_name: contact-service
+    working_dir: /src
+    command: >
+      bash -c "npm install && node ace migration:run && npm run start"
+    volumes:
+      - .:/src
+    networks:
+      - main_network
+  db:
+    image: mysql
+    command: --default-authentication-plugin=mysql_native_password
+    ports:
+      - 3306:3306
+    restart: always
+    environment:
+      MYSQL_DATABASE: swarley
+      MYSQL_ALLOW_EMPTY_PASSWORD: "yes"
+    networks:
+      - main_network
+
+networks:
+  main_network:
+    name: elastic_external_network
```

Comment:
What's with the name?

Reply:
Was absolutely random

Reply:
Let's just name it `swarley_network`. I don't think the ELK installation will run on the same network, since they're communicating via the host's ports.

Reply:
Although, that means the other service would also have to be on the `swarley_network`